app/services/solidus_bolt/orders/track_shipment_service.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+module SolidusBolt
+  module Orders
+    class TrackShipmentService < SolidusBolt::BaseService
+      attr_reader :transaction_reference, :shipment
+
+      def initialize(transaction_reference:, shipment:)
+        @transaction_reference = transaction_reference
+        @shipment = shipment
+        super
+      end
+
+      def call
+        track_shipment
+      end
+
+      private
+
+      def track_shipment
+        options = build_options
+        handle_result(
+          HTTParty.post(
+            "#{api_base_url}/#{api_version}/merchant/track_shipment",
+            options
+          )
+        )
+      end
+
+      def build_options
+        order = shipment.order
+        {
+          body: {
+            transaction_reference: transaction_reference,
+            tracking_number: shipment.tracking,
+            carrier: shipment.shipping_method.name,
+            items: order.bolt_items,
```

Comment:
This is not right, the shipped items should be the items in the shipment, not the items in the order.

Reply:
Moved this to the `shipment_decorator`.